src/sider/arbeidsuforhet/ForhandsvarselSendt.tsx

```diff
@@ -17,6 +40,24 @@ export const ForhandsvarselSendt = () => {
       ) : (
         <ForhandsvarselAfterDeadline />
       )}
+      <Box background="surface-default" padding="3" className="mb-2">
```

Comment:
Til diskusjon med Peter: Hadde det gitt mening å slå sammen denne og `ForhandsvarselBefore/AfterDeadline`? Siden nå står fanene uten noen overskrift. 
<img width="591" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/79630b68-651d-41eb-b312-83ac81998b20">

Reply:
Støttes. Tror de fint kan slås sammen, men må få noen gode designøyne på hvordan det kan se pent ut med Se hele brevet-knappen over de to fanene.